ITC205_BaseLine_Code/src/ReturnBookUI.java

```diff
@@ -3,17 +3,17 @@
 
 public class ReturnBookUI {
 
-	public static enum UI_STATE { INITIALISED, READY, INSPECTING, COMPLETED };
+	public static enum UIState { INITIALISED, READY, INSPECTING, COMPLETED };
 
 	private ReturnBookControl control;
 	private Scanner input;
-	private UI_STATE state;
+	private UIState state;
 
 	
 	public ReturnBookUI(ReturnBookControl control) {
 		this.control = control;
 		input = new Scanner(System.in);
-		state = UI_STATE.INITIALISED;
+		state = UIState.INITIALISED;
 		control.setUI(this);
 	}
 

```

Comment:
D13 - Line 48 & 50:

String ans = input("Is book damaged? (Y/N): ");
 				boolean isDamaged = false;
 				if (ans.toUpperCase().equals("Y")) {

Change ans -> answer